packages/web-components/src/components/va-memorable-date/test/va-memorable-date.e2e.ts

```diff
@@ -87,7 +87,9 @@ describe('va-memorable-date', () => {
 
   it('renders hint text', async () => {
     const page = await newE2EPage();
-    await page.setContent('<va-memorable-date label="Label" hint="hint text" required month-select ></va-memorable-date>');
+    await page.setContent(
+      '<va-memorable-date label="Label" hint="hint text" required month-select ></va-memorable-date>',
+    );
```

Comment:
There are 2 commits. The first one applies linting cleanup and the second contains the actual code changes.